specification/eventgrid/data-plane/Microsoft.ApiManagement/stable/2018-01-01/examples/apireleaseDeleted.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "95015754-aa51-4eb6-98d9-9ee322b82ad7",
+  "topic": "/subscriptions/{subscription-id}/resourceGroups/{your-rg}/providers/Microsoft.ApiManagement/service/{your-APIM-instance}",
+  "subject": "/apis/{apiId}/releases/{releaseId}",
+  "data": {
+    "resourceUri": "/subscriptions/subscription-id}/resourceGroups/{your-rg}/providers/Microsoft.ApiManagement/service/{your-APIM-instance}/apis/{apiId}/releases/{releaseId}"
+  },
+  "eventType": "Microsoft.ApiManagement.APIReleaseDeleted",
```

Comment:
Just want to check - are the event types still using "API" or were they also updated to "Api" to match the event names?

Reply:
/cc @KacieKK

Reply:
My mistake - I see that the event names were updated, but the event descriptions were left in the API casing. Please disregard my question.